pkg/image/api/helper.go

```diff
@@ -128,6 +128,18 @@ func (r DockerImageReference) AsRepository() DockerImageReference {
 	return r
 }
 
+// DaemonMinimal clears defaults that Docker assumes.
```

Comment:
right, so it seems bad to be stripping the names down since they become ambiguous.